src/test/java/testresults/annual_certificates/TestPostTestResultsAnnualCertificateGenerationHgv.java

```diff
@@ -241,7 +242,8 @@ public TestPostTestResultsAnnualCertificateGenerationHgv(String name, String tes
         this.testCode = testCode;
     }
 
-    @WithTag("annual_certificates")
+//    @WithTag("annual_certificates")
```

Comment:
Same here, is commented out code required?

Reply:
Removed